scripts/helpers/restyle-diff.sh

```diff
@@ -41,6 +41,17 @@ restyle-paths() {
 cd "$CHIP_ROOT"
 declare -a paths=()
 
-readarray -t paths < <(git diff --ignore-submodules --name-only "${1:-master}")
```

Comment:
Or we could just tell people to install a modern bash on OSX, honestly, via macports or brew?  We tell them to install all sorts of other stuff already to build CHIP.

(For the record, I am 99% sure the author of this code was on Mac, and so were some of the reviewers, but they all had modern bash installed.)

Reply:
I don't believe we require anything to be manually installed other than X Code & openssl. The rest are brought in via bootstrap in an as seamless a way as we can manage.

The OpenSSL dependency has actually tripped up quite a few people; it would be good to make that more seamless as well.

Reply:
.. especially if such dependencies are managed in a manual way

(I draw a distinction between things that must be separately documented, and things that are automated. I don't think we should distribute bash, however... we already require python and if we need scripts to work cross-platform (**Windows** ..) that's what we should use.

Reply:
To be clear, I have no objections to making this change.